corehq/apps/hqwebapp/static/hqwebapp/js/main.js

```diff
@@ -28,6 +29,23 @@ var eventize = function (that) {
     return that;
 };
 
+// This is copy of csrf ajax utilities from hqwebapp/js/ajax_csrf_setup.js
```

Comment:
just because i think DRY is very important. for example, if 3 months from now we fix a bug in this function we're going to have to remember that it lives in two places. whereas if we explicitly update the templates/dependencies then we can be sure that the logic only lives in one place and whenever we make a fix (by adding the import) it should work from then onwards.

that said, i think using something like bower or requirejs might be the best answer, but i'm not totally familiar with exactly how to do that.